docs/rules/object-key-newlines.md

```diff
@@ -0,0 +1,69 @@
+# Enforce placing object keys on separate lines (object-key-newlines)
+
+While formatting preferences are very personal, a number of style guides require that
+object keys be placed on separate lines for better readability.
+
+## Rule Details
+
+This rule aims to maintain consistency of newlines between object keys.
+
+Examples of **correct** code for this rule:
```

Comment:
The conventional order is **incorrect** code first and **correct** code second.

As people say about hammers and nails, so also linters and errors, I guess.